src/plugins/intel_gpu/src/kernel_selector/cl_kernels/reorder_weights.cl

```diff
@@ -313,11 +315,11 @@ inline uint FUNC(get_output_index)(uint g, uint o, uint i, uint z, uint y, uint
 #elif defined OUTPUT_LAYOUT_OS_IS_ZYX_OSA4_ISA8_OSV8_ISV4_SWIZZLED_BY_4
     return GET_FILTER_OS_IS_ZYX_OSA4_ISA8_OSV8_ISV4_SWIZZLED_BY_4_INDEX(OUTPUT, o, i, z, y, x);
 #elif defined OUTPUT_LAYOUT_OS_IS_YX_ISV16_OSV16
-    return GET_FILTER_OS_IS_YX_ISV16_OSV16_INDEX(OUTPUT, o, i, y, x, SUB_GROUP_SIZE);
```

Comment:
could you explain why cldnn kernels were OK with this code?

Reply:
In the ref(cldnn) case, weight reorder is `os_is_yx_osv16` to `os_is_yx_osv16` and result will be correct regardless of correctness of index calculation.

Reply:
Do you mean that cldnn was not using isv16_osv16 format for weights in all cases? If they have used it for any reason, it must have failed for accuracy. Am I correct?

by the way, where was the SUB_GROUP_SIZE defined as 1, instead of 16?

Reply:
`fusings_gpu/conv_fp32_prelu_eltwise.basic_sum/3` use isv16osv16 and weight reorder proceeded reordering from `isv16osv16` to `isv16osv16` in both ref and opt when using igpu(cldnn). In DG2(onednn), required weight reorder is `isv16osv16` to `oiyx` and it cause testcase failure.

Reply:
I printed SUB_GROUP_SIZE in the kernel and the value was 1. I don't know where it defined.